.github/workflows/ci-e2e-badger.yaml

```diff
@@ -41,6 +43,12 @@ jobs:
             ;;
         esac
 
+    - uses: ./.github/actions/verify-metrics-snapshot
+      if: matrix.version == 'v2'
+      with:
+        snapshot: metrics_snapshot_badger
```

Comment:
is the snapshot name supposed to have `.txt` extension? Is the key?

I am seeing this in the workflow run:
```
No files were found with the provided path: ./.metrics/metrics_snapshot_memory.txt. No artifacts will be uploaded.
```